src/ext.rs

```diff
@@ -1102,6 +1110,12 @@ impl UdpSocketExt for UdpSocket {
                mreq)
     }
 
+    #[cfg(target_os = "nto")]
+    fn leave_multicast_v6(&self, _: &Ipv6Addr, _: u32)
+                          -> io::Result<()> {
+        Err(io::Error::new(io::ErrorKind::Unsupported, "Not supported by QNX Neutrino"))
+    }
+                    
```

Comment:
There appears to be trailing whitespace added here

Reply:
Fixed -- thanks! You have good eyes.